packages/adapter-netlify/index.js

```diff
@@ -1,57 +1,63 @@
-'use strict';
-
 import { existsSync, readFileSync, copyFileSync, writeFileSync, renameSync } from 'fs';
 import { dirname, resolve } from 'path';
 import toml from 'toml';
 import { fileURLToPath } from 'url';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
-export default async function adapter(builder) {
-	let netlify_config;
-
-	if (existsSync('netlify.toml')) {
-		try {
-			netlify_config = toml.parse(readFileSync('netlify.toml', 'utf-8'));
-		} catch (err) {
-			err.message = `Error parsing netlify.toml: ${err.message}`;
-			throw err;
+export default function () {
```

Comment:
is prettier running on this file? I wouldn't have expected a space after `function` here

Reply:
I don't know whether that's something that changed at some point, but no, this is what Prettier currently does.

Reply:
prettier adds a space after `function`. it removes the space after the function's _name_, if there is one